File: nomor 2/golang/ex10.go
package main

import (
	"fmt"
	"math/rand"
)

var telco = []string{"Telkom", "Indosat", "XL", "Verizon", "AT&T", "Nippon", "Vodafone", "Orange", "KDDI", "Telefonica", "T-Mobile"}

func findCompany(array []string, input int) {
	for i := 0; i < len(array); i++ {
		if array[i] == array[input] {
			fmt.Printf("Company Found: %s\n", array[input])
			break
		}
		fmt.Printf("Searching Company... %d\n", i+1)
	}
}
func main() {
	var company int = rand.Intn(len(telco))
	findCompany(telco, company)
}

File: nomor 2/kotlin/ex10.kt
import kotlin.random.Random

val telco = listOf("Telkom", "Indosat", "XL", "Verizon", "AT&T", "Nippon", "Vodafone", "Orange", "KDDI", "Telefonica", "T-Mobile")

fun findCompany(array: List<String>, input: Int) {
    for (i in array.indices) {
        if (array[i] == array[input]) {
            println("Company Found ${array[input]}")
            break
        }
        println("Searching Company...${i+1}")
    }
}

fun main() {
    val company = Random.nextInt(telco.size)
    findCompany(telco, company)
}
